File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao02.py
while True:
    nome_usuario = input("Digite o nome do usuário: ")
    senha_usuario = input("Digite a senha do usuário: ")
    if nome_usuario != senha_usuario:
        print("A senha e o usuário são válidos.")
        break
    else:
        print("Senha e usuário inválidos.")

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Kotlin/questao02.kt
fun main(){
    while(true){
        val nomeUsuario = "Paulo"
        val senhaUsuario = "123"
        if(nomeUsuario != senhaUsuario){
            println("O usuário e a senha são válidos")
            break
        }else{
            println("Usuário e senha invalidos")
        }
    }
}